orchestrator/orchestratorAutomationAIC21-10.ps1

```diff
@@ -0,0 +1,209 @@
+#Requires -RunAsAdministrator
+
+<#
+
+.SYNOPSIS
+   Makes aifabric related changes to orchestrator web.config to enable aifabric installation and access.
+.DESCRIPTION
+   Add entries in orchestrator web.config(if not exists) for orchestrator internal IDP and aifabric access from robot and orchestrator.
+   Removes cache to allow access to new controllers and resets iis to load new values.
+.NOTES
+    Name: ./orchestratorAutomation.ps1
+    Author: AIFabric Team
+    Pre-Requisites: script has to be executed throuh powershell in Administrator mode & before running script set execution policy to RemoteSigned by running "Set-ExecutionPolicy RemoteSigned"
+.EXAMPLE
+    If aifabric is available at ww.xx.yy.zz:port, command to run would be
+    .\orchestratorAutomation.ps1 -aifUrl ww.xx.yy.zz:port
+
+    If ai-app is accessed via domain instead of IP:PORT combo, then enable domainBasedAccess to true
+    .\orchestratorAutomation.ps1 -aifUrl "aif-sahil-aks.westeurope.cloudapp.azure.com"
+
+    If Orchestrator Installation Path has to be specified,
+    .\orchestratorAutomation.ps1 -aifUrl ww.xx.yy.zz -config "C:\Program Files (x86)\UiPath\Orchestrator"
+
```

Comment:
Fix the example in Notes.

Reply:
@Mdirfan6 I can still see examples where aifUrl is provided but storageUrl is not. Both are mandatory going forward.

Reply:
Updated changes as mentioned.